### DISCUSSION TITLE
Sobre blogs e monetização

### DISCUSSION THREAD

**Discussion by devigor** [frontendbr/forum#1804] [Discussão]
Em tempos de pandemia muitas pessoas perderam os empregos, inclusive eu, o que me fez pensar alguma forma de se levantar um dinheiro extra para poder sobreviver nesses tempos! E com isso cheguei a um pensamento de monetização de artigos, o Medium eu sei que faz isso, porém queria fazer isso dentro do meu blog pessoal. 
Alguém já fez algo do tipo e queira compartilhar? 
E quem foi afetado pela pandemia, o que tem feito para levantar uma grana?

**Comment by lucasferreiralimax** [comment]
Até hoje eu tento monetizar meu projeto [Currículo Grátis](http://curriculo-gratis.web.app)
Olha que tem um fluxo de acessos faz um bom tempo, mas o google não aceitou adsense.
Também me encontro desempregado procurando uma empresa para colaborar e ajudar.

**Comment by kazzkiq** [comment]
@lucasferreiralimax já cogitou a possibilidade de fechar patrocínio direto com empresas?

Há muitos anos atrás eu e um sócio seguimos por esse caminho em um portal de notícias regional, e acabou dando mais certo que AdSense. As vezes pode ser uma estratégia útil no seu caso também.

**Comment by lucasferreiralimax** [comment]
> @lucasferreiralimax já cogitou a possibilidade de fechar patrocínio direto com empresas?
> 
> Há muitos anos atrás eu e um sócio seguimos por esse caminho em um portal de notícias regional, e acabou dando mais certo que AdSense. As vezes pode ser uma estratégia útil no seu caso também.

Quais empresas teriam interesse em divulgar em um gerador de cv grátis?
Eu ainda tento o adsense quem sabe um dia eles aprovam

**Comment by ViniciusGularte** [comment]
@devigor  o medium envia pagamentos pelo stripe pelo que sei, e ainda não tem suporte no brasil :disappointed:  , ja pensou em algo como  o [https://www.buymeacoffee.com/](https://www.buymeacoffee.com/) ?
